solr.cfg

```diff
@@ -39,12 +39,13 @@ java_opts =
   -server
   -Xms${settings:solr-min-ram}
   -Xmx${settings:solr-max-ram}
+
 index =
-    name:allowedRolesAndUsers   type:string stored:false multivalued:true
```

Comment:
@maethu could you elaborate why we have to store those values? Why would we want to return the "default" or the "allowedRolesAndUsers" field in our search results? Is this even related to the new feature?